nxdrive/metrics.py

```diff
@@ -0,0 +1,40 @@
+from logging import getLogger
+from queue import Empty, Queue
+from typing import Any, Dict
+
+from .engine.workers import Worker
+from .exceptions import ThreadInterrupt
+
+log = getLogger(__name__)
+
+
+class DriveMetrics(Worker):
```

Comment:
You can remove "Drive" from the class name.

Reply:
WDYT of using a `PollWorker` directly? It will save us having to implement all that logic. See https://github.com/nuxeo/nuxeo-drive/blob/master/nxdrive/poll_workers.py for examples.

We could think of a worker with a delay of 5 minutes, maybe more.